ui/pages/confirmation/templates/add-ethereum-chain.js

```diff
@@ -181,7 +181,14 @@ function getState(pendingApproval) {
   return {};
 }
 
-function getValues(pendingApproval, t, actions, history) {
+function getValues(
+  pendingApproval,
+  t,
+  actions,
+  history,
+  _setInputState,
+  unnaprovedTxsCount,
```

Comment:
Should we put this inside an object such as `state` perhaps so we can group any additional relevant state in future without adding more arguments to the method?